docs/developer-documentation.md

```diff
@@ -2,28 +2,31 @@
 
 ## Overview
 Zulip Terminal uses [Zulip's API](https://zulipchat.com/api/) to store and retrieve all the information it displays. It has an [MVC structure](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller) overall. Here is a description of some of its files:
- 
-| File/Folder                                           | Description                                   |
-| ----------------------------------------------------- | --------------------------------------------- |
-|zulipterminal/                                         | Root Folder - Contains all the source files   |
-|              ui.py                                    | Controls where each component is displayed    |
-|              core.py                                  | Runs the app and controls data flow into View |
-|              model.py                                 | Fetches and stores data retrieved from server |
-|              helper.py                                | Helper functions used at multiple places      |
-|              config.py                                | Stores keybindings along with what they do    |
-|              ui_tools                                 | Has all the UI elements displayed by View     |
-|              cli/run.py                               | Runs the app                                  |
+
+| File/Folder                                           | Description                                      |
+| ----------------------------------------------------- | ---------------------------------------------    |
+|zulipterminal/                                         | Root Folder - Contains all the source files      |
+|              cli/run.py                               | Marks the entry point into the application       |
```

Comment:
None of these indents are rendered, so perhaps we could have multiple tables for each main folder? Or references to the main folders in the text?